src/components/tags/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file Implements Tags component to display a list of tags
+ */
+import React, {useEffect, useState} from "react";
+import './tags.css';
+import Tag from "./tag";
+import * as likeService from "../../services/likes-service";
+import * as dislikeService from "../../services/dislikes-service";
+import * as tagService from '../../services/tags-service';
+import * as authService from "../../services/auth-service";
+
+const Tags = ({tags = [], refreshTags}) => {
+    const [profile, setProfile] = useState(undefined);
+    useEffect(async ()=> {
+        try {
+            const user = await authService.profile();
+            if (user) {
+                setProfile(user);
+            }
+        } catch (e) {
+        }
+    }, []);
+
+    /**
```

Comment:
Is this a thing?  Do we need this method?

Reply:
Keeping for now if that's alright (using for my-likes page on explore page). Will remove these functions later when we have more of the tags/tuits functionality working; I just like having it here for reference while building the other features.

Reply:
Sounds good!